KaylaKornelis-app/src/main/java/edu/KaylaKornelis/advancedjava/Assignment1/Book.java

```diff
@@ -0,0 +1,47 @@
+package edu.KaylaKornelis.advancedjava.Assignment1;
+
+public class Book{
+
+    //Attributes of a Book
+    private String title;
+    private String author;
+    private String genre;
+
+    /**
+     * Create a new Book using the following attributes
+     *
+     * @param title - the title of the chosen book
+     * @param author - the name of the Author who wrote the book
+     * @param genre - the genre the book is categorized by
+     *
+     */
+    public Book(String title, String author, String genre){
+        this.title = title;
+        this.author = author;
+        this.genre = genre;
+    }
+
```

Comment:
Javadoc all methods (even getters and setters) for consistency 
(If you haven't already you'll find varying opinions about this, but for me the consistency makes the code more readable. )